tests/Microsoft.Fx.Portability.Tests/BreakingChangeTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace Microsoft.Fx.Portability.Tests
+{
+    public class BreakingChangeTests
+    {
+        [Fact]
+        public void EqualityTests()
+        {
+            var breakingChangeSame1 = new BreakingChange { Id = "id1" };
+            var breakingChangeSame2 = new BreakingChange { Id = "id1" };
+            var breakingChangeDifferent1 = new BreakingChange { Id = "ID1" };
+            var breakingChangeDifferent2 = new BreakingChange { Id = "id2" };
```

Comment:
I think I did - look at line 19 and line 28. Or are you referring to a specific test like:

```csharp
Assert.False(string.Equals(breakingChangeSame1.Id, breakingChangeDifferent1.Id, StringComparison.Ordinal));
```